workshops/build-a-binderhub/workshop-presentations/zero-to-binderhub.md

```diff
@@ -314,100 +365,39 @@ aks-nodepool1-97000712-0   Ready    agent   19m   v1.9.11
 
 ### Setting up Helm
 
-Adapted from [Zero-to-JupyterHub: Setting up and Securing Helm](https://zero-to-jupyterhub.readthedocs.io/en/latest/setup-helm.html).
+Adapted from [Zero-to-JupyterHub: Setting up Helm](https://zero-to-jupyterhub.readthedocs.io/en/latest/kubernetes/setup-helm.html).
 
-Helm is the package manager for Kubernetes and is used for: installing, upgrading and managing applications on a Kubernetes cluster.
+Helm is the package manager for Kubernetes and is used for installing, upgrading and managing applications on a Kubernetes cluster.
 Helm packages are called _charts_.
-
-Helm has two parts: a client (`helm`) and a server (`tiller`).
-Tiller runs inside your Kubernetes cluster as a pod in the `kube-system` namespace.
-Tiller manages _releases_ (installations) and _revisions_ (versions) of charts deployed on the cluster.
-When you run a `helm` command, the local Helm client sends instructions to `tiller` in the cluster which in turn makes the requested changes.
+Helm manages _releases_ (installations) and _revisions_ (versions) of charts deployed on the cluster.
 
 > **Did you know?:** Kubernetes is Greek for "captain" or "helmsman".
 > In case you haven't noticed the nautical theme!
 
-#### 1. Setup a `ServiceAccount` for `tiller`
-
-When you (a human) accesses your Kubernetes cluster, you are authenticated as a particular **User Account**.
-Processes in containers running in _pods_ are authenticated as a particular **Service Account**.
-[More details](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/).
-
-```
-kubectl create serviceaccount tiller --namespace kube-system
-```
-
-:vertical_traffic_light: :vertical_traffic_light: :vertical_traffic_light: :vertical_traffic_light: :vertical_traffic_light:
-
-#### 2. Give the `ServiceAccount` full permissions to manage the cluster
-
-This step enables Role Based Access Control (RBAC) so Kubernetes can secure which pods/users can perform what kind of actions on the cluster.
-If RBAC is disabled, **all pods are given `root` equivalent permission on all the Kubernetes nodes and the cluster itself.**
-This can leave the cluster vulnerable to attacks.
-See [Project Jupyter's docs](https://zero-to-jupyterhub.readthedocs.io/en/latest/security.html#use-role-based-access-control-rbac) for more details.
-
-```
-kubectl create clusterrolebinding tiller --clusterrole cluster-admin --serviceaccount=kube-system:tiller
-```
-
-:vertical_traffic_light: :vertical_traffic_light: :vertical_traffic_light: :vertical_traffic_light: :vertical_traffic_light:
-
-:question: :question: :question: :question: :question:
-
-#### 3. Initialise `helm` and `tiller`
+#### 1. Verify Helm
```

Comment:
In jupyterhub resource, they have installation step before verification.

Reply:
Yes, they do. I normally run this workshop using the Azure Cloud Shell however, which has a lot of tools preinstalled so I don't have to worry about getting people setup before the day.